doc/languages-frameworks/python.section.md

```diff
@@ -1471,6 +1475,9 @@ with python311Packages;
 buildPythonPackage rec {
   name = "mypackage";
   src = ./path/to/package/source;
+  venvDir = "./.venv"; # used as the editable install destination
```

Comment:
A very possible user-error is the following change:

```suggestion
  venvDir = ./.venv; # used as the editable install destination
```

This will be turned into a nix store path, which may cause cryptic errors. Could you add a check against this? Considering the store prefix `/nix/store` is configurable, i think checking the type is the way to go.

Reply:
I don't get this value in the nixlang anywhere. It just gets transformed into an environment variable using the normal derivation mechanism.

If the path doesn't exist then the instantiation will fail with a read error. Unfortunately I don't think there's any way to mitigate this error, unless you can tell me where to put the check. It can't be in the shell script since we'll fail before that gets a chance to run if it doesn't exist.

Reply:
Maybe check if it is writeable, then emit a helpful message

Reply:
I don't think our code can run before the check and error.